```
apps/issues/models.py:279:1: E402 module level import not at top of file
apps/issues/models.py:337:13: E122 continuation line missing indentation or outdented
apps/issues/models.py:338:13: E122 continuation line missing indentation or outdented
apps/issues/models.py:339:13: E122 continuation line missing indentation or outdented
apps/issues/models.py:340:13: E122 continuation line missing indentation or outdented
apps/issues/views.py:78:1: E402 module level import not at top of file
apps/issues/views.py:195:1: E402 module level import not at top of file
```

Few things to fix still...

--- a/apps/issues/models.py
+++ b/apps/issues/models.py
@@ -1,6 +1,7 @@
 from django.db import models
 from github3 import login as gh_login
 from django.core.exceptions import ValidationError
+from django.core.urlresolvers import reverse
 from django.conf import settings
 import urlparse
 
@@ -276,8 +277,6 @@ class Issue(models.Model):
     def __str__(self):
         return self.url
 
-from django.core.urlresolvers import reverse
-
 
 class Subscriber(models.Model):
 
@@ -331,14 +330,20 @@ email address at %s?token=%s" % (
         from django.core.mail import send_mail
         self.token = self.subscriptions_token(issue)
 
-        send_mail("ACME Issues: Confirm issue subscription",
-                  "Please click here to confirm your subscription to \
-the issue '%s': %s?token=%s&issue=%d>" % (
+        message = "Please click here to confirm your subscription to \
+the issue '%s': %s?token=%s&issue=%d>"
+
+        message %= (
             issue.name,
             reverse("confirm_subscription"),
             self.token,
-            issue.id),
-            "fries2@llnl.gov", [self.email])
+            issue.id
+        )
+
+        send_mail("ACME Issues: Confirm issue subscription",
+                  message,
+                  "fries2@llnl.gov",
+                  [self.email])
         self.save()
 
     def __str__(self):

--- a/apps/issues/views.py
+++ b/apps/issues/views.py
@@ -6,6 +6,8 @@ from django.http import (
     HttpResponseNotAllowed
 )
 from django.template import RequestContext, loader
+from django.core.urlresolvers import reverse
+from templatetags.issues import render_question_tree
 from models import *
 import json
 
@@ -74,8 +76,6 @@ def make_issue(request):
 
     user.subscribe(issue)
     return HttpResponse("")
-
-from django.core.urlresolvers import reverse
 
 
 def confirm_email(request):
@@ -191,8 +191,6 @@ def show_question(request, source):
             "issues/questions.html",
             {"source": s, "root_questions": related}
         ))
-
-from templatetags.issues import render_question_tree
 
 
 @post_only

